ui/src/mixins/search-filter.js

```diff
@@ -322,6 +334,21 @@ export default {
       }
 
       return options
+    },
+
+    mx_registerSelectedOptionsEvent () {
```

Comment:
Pra que precisou disto?

Reply:
![Screenshot 2024-02-20 at 16 50 37](https://github.com/bildvitta/asteroid/assets/19765074/392b65b8-866f-4883-8815-8d72235d4c76)

Lembra desse comentário? Eu tirei do watch geral e joguei dentro desse método só para adicionar o watch das opções selecionadas se for lazy loading.